### ISSUE TITLE
Improvement Suggestions

### ISSUE THREAD

**Issue by fuerbringer** [GianlucaScarciolla/gianlucascarciolla.github.io#3]
As discussed earlier I'd recommend improving your site. Consider the following points:

### General

- [x] Favicon missing

### On https://scarciolla.info/#s1

Instead of `Stuff is coming soon!` list...
- [x] Your profession
- [x] Maybe one or two projects you've done
- [x] Maybe hobbies

### On https://scarciolla.info/#s2

I'd list following projects (Maybe in brackets the technologies used), linked ofcourse:
Also rename to `Projects I've contributed to`.

- [x] Your CT Stuff
- [x] https://github.com/fuerbringer/M133-Abschluss
- [x] https://github.com/PROGRADE-Tech/Tarchivebot

### On https://scarciolla.info/#s3

Basically a more detailed version of #s1

### On https://scarciolla.info/#s4

You linked your mail wrong.. (Says `hgmail.com` instead of `gmail.com`)

**Comment by GianlucaScarciolla** [comment]
thank you for helping me to improve :)
